spectrome/forward/runforward.py

```diff
@@ -1,47 +1,8 @@
 """ running the ntf over a range of frequencies."""
-from ..forward import network_transfer as nt
+from ..forward import network_transfer_P as nt
```

Comment:
Does `forward.network_transfer_P` exist?

Reply:
So I had it at some point just because I was playing with some stuff but removed it later on. I don't have `network_transfer_P` in the final version that I sent you (right?). Do you want me to remove that commit from the history? 

So `network_transfer_local_alpha` is the naming convention you had. I think we should discuss on how we want to keep the names. I think we can remove `network_transfer_function` and replace it with `network_transfer_local_alpha`. If you like this idea, I will go ahead with it. Let me know.

Reply:
From the looks of the code `from ..forward import network_transfer_P` is in the code of the pull request, if that function doesn't exist in `forward`, what are we importing? Does `nt` run in your local copy? The import should fail if `forward` doesn't have the function. 

Let's discuss naming in the zoom meeting.